.gitignore

```diff
@@ -24,12 +24,15 @@ Makefile.in
 /perf.data
 /perf.data.old
 /stamp-h1
+/CMakeCache.txt
```

Comment:
In general I find it weird that we use absolute paths starting with `/` in this file. Isn't it usually all relative?